File: Python/1012 - Área.py
n = input().split()

tri = float(n[0])*float(n[2])/2
cir = float(n[2])**2 * 3.14159
tra = (float(n[0]) + float(n[1])) * float(n[2])/2
qua = float(n[1])**2
ret = float(n[0])*float(n[1])

print(f"TRIANGULO: {'%.3f' % tri}")
print(f"CIRCULO: {'%.3f' % cir}")
print(f"TRAPEZIO: {'%.3f' % tra}")
print(f"QUADRADO: {'%.3f' % qua}")
print(f"RETANGULO: {'%.3f' % ret}")

File: Ruby/1012 - Área.rb
abc = gets.split(' ')

a = abc[0].to_f
b = abc[1].to_f
c = abc[2].to_f

tri = (a*c)/2
cir = c**2 * 3.14159
tra = (a+b) * c/2
qua = b**2
ret = a*b

puts "TRIANGULO: #{format("%.3f", tri)}"
puts "CIRCULO: #{format("%.3f", cir)}"
puts "TRAPEZIO: #{format("%.3f", tra)}"
puts "QUADRADO: #{format("%.3f", qua)}"
puts "RETANGULO: #{format("%.3f", ret)}"
